src/worker.jl

```diff
@@ -2,6 +2,19 @@ function worker_init(f::File, options::Dict)
     project = WorkerSetup.LOADER_ENV[]
     return lock(WorkerSetup.WORKER_SETUP_LOCK) do
         return quote
+            # issue #192
+            # Malt itself uses a new task for each `remote_eval` and because of this, random number streams
+            # are not consistens across runs even if seeded, as each task introduces a new state for its
+            # task-local RNG. As a workaround, we use feed all `remote_eval` requests through these channels, such
+            # that the task executing code is always the same.
+            var"__stable_execution_task_channel_out" = Channel()
+            var"__stable_execution_task_channel_in" = Channel() do chan
```

Comment:
Do these need to be `var""`s?

Reply:
Also, should be `const` if possible.

Reply:
We only eval user code into notebook modules right? Then no, at first I thought these would live side-by-side